model/project.go

```diff
@@ -1334,7 +1334,7 @@ func (p *Project) FindDistroNameForTask(t *task.Task) (string, error) {
 // FindLatestVersionWithValidProject returns the latest mainline version that
 // has a valid project configuration. It also returns the intermediate and final
 // project configurations.
```

Comment:
Can we mention what it means to request a `preGeneration` project YAML?

Reply:
Added